src/containers/Auth/SignUp.jsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import * as Yup from "yup";
+import AppFormField from "../../components/Form/AppFormField";
+import SubmitButton from "../../components/Form/SubmitButton";
+import AppForm from "../../components/Form/AppForm";
+import Grid from "@material-ui/core/Grid";
+import Block from "../../components/Block/Block";
+import { useHistory } from "react-router-dom";
+import Button from "@material-ui/core/Button";
+import { makeStyles } from "@material-ui/core/styles";
+
+const validationSchema = Yup.object().shape({
+  email: Yup.string().required().email().label("Email"),
+  password: Yup.string().required().label("Password"),
+});
+
+const useStyles = makeStyles((theme) => ({
+  button: {
+    padding: theme.button.padding,
+  },
+}));
+
+const SignUp = () => {
+  const history = useHistory();
+  const classes = useStyles();
+
+  return (
+    <Block lg={6} md={8} sm={10} xs={11}>
+      <AppForm
+        initialValues={{ fname: "", lname: "", email: "", password: "" }}
```

Comment:
Feel free to use easily readable names, like firstName and lastName.

Reply:
I think this is really important, never underestimate how easily a code can or cannot be read, since you might be the maintainer of your code after months or years. (I am referring to work projects obviously)